src/content/registry-list.json

```diff
@@ -177,7 +177,6 @@
       "BBOI",
       "TC",
       "AFRINIC",
-      "ARIN-WHOIS",
```

Comment:
This should be changed to `ARIN` as it it is still being mirrored according to the qna, and the other fields uses `ARIN` instead of `ARIN-WHOIS`.
https://www.gin.ntt.net/support-center/policies-procedures/routing-registry/

Reply:
`ARIN` is already on the list (line 176).

Reply:
I missed that, thanks for the correction.